src/components/header/_header.scss

```diff
@@ -110,29 +109,41 @@
 
     &--neutral & {
         &__top {
-            background: var(--ons-color-header-masthead-neutral);
+            .ons-icon--logo {
+                display: block;
+                .ons-icon--logo__group--text,
+                .ons-icon--logo__group--primary {
+                    fill: var(--ons-color-black);
+                }
+            }
+        }
+
+        &__org-logo-link:focus {
+            outline: none;
+            background-color: var(--ons-color-focus);
+            .ons-icon--logo {
+                color: var(--ons-color-text-link-focus);
+            }
         }
 
         &__grid-top {
             a {
-                color: var(--ons-color-service-links);
+                color: var(--ons-color-black);
                 &:hover {
-                    text-decoration: underline solid var(--ons-color-service-links) 3px;
+                    text-decoration: underline solid var(--ons-color-header-masthead-neutral) 3px;
+                }
+                &:focus {
+                    color: var(--ons-color-black);
```

Comment:
These lines aren't adding anything because the attribute they are setting is already set to black so they can be removed